dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/ClientWithNoCdi.java

```diff
@@ -36,18 +34,24 @@
 import componenttest.custom.junit.runner.FATRunner;
 import componenttest.custom.junit.runner.Mode;
 import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.MicroProfileActions;
+import componenttest.rules.repeater.RepeatTests;
 import componenttest.topology.impl.LibertyServer;
-import componenttest.topology.utils.FATServletClient;
 import componenttest.topology.utils.HttpRequest;
-
 import io.openliberty.microprofile.telemetry.internal_fat.apps.clientnocdi.ClientTriggeringServlet;
+
 @Mode(TestMode.FULL)
 @RunWith(FATRunner.class)
 public class ClientWithNoCdi {
 
     public static final String SERVER_NAME = "Telemetry10JaxWithLogging";
     public static final String NO_CDI_APP_NAME = "clientNoCDI";
 
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.MP60, MicroProfileActions.MP61)
+                    .andWith(FeatureReplacementAction.BETA_OPTION().fullFATOnly());
+
     @TestServlets({
                     @TestServlet(contextRoot = NO_CDI_APP_NAME, servlet = ClientTriggeringServlet.class),
     })
```

Comment:
As a follow-up: this should be removed - `ClientTriggeringServlet` is not a test servlet.